docs/getstarted.md

```diff
@@ -1,110 +1,169 @@
 (getstarted-page)=
 
-# Getting started
+# Get started
+
+If you're new to Nextflow, [take a look at the tutorials on this page](https://www.nextflow.io/blog/2023/learn-nextflow-in-2023.html#nf-core) first. This guide will help you get started with using Nextflow. It covers: 
+
+- Requirements
+- Installation
+- Your first script
+
+We also recommend taking a look at these pages once you've installed Nextflow.
+
+- {ref}`config-page`
+- {ref}`workflow-page`
+- {ref}`operator-page`
 
 (getstarted-requirement)=
 
 ## Requirements
 
-Nextflow can be used on any POSIX compatible system (Linux, macOS, etc). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 21)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed.
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 21)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed. You can check what version you have using the following command:
+
+```bash
+java -version
+```
 
-For the execution in a cluster of computers, the use of a shared file system is required to allow the sharing of tasks input/output files.
+We recommend that you install Java through [SDKMAN!](https://sdkman.io/), and that you use the latest LTS version of Temurin. See [this website](https://whichjdk.com/) for more information.
 
-Nextflow can also be run on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
+To install Java with SDKMAN:
```

Comment:
@bentsherman, could you validate that this is all the information we need here - there isn't anything missing for other operating systems etc. Thank you!

Reply:
It is the same across the board